client/lib/preferences-helper/PreferenceList.js

```diff
@@ -0,0 +1,69 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { partial, isEmpty } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import { getAllRemotePreferences } from 'state/preferences/selectors';
+import { savePreference } from 'state/preferences/actions';
+
+class PreferenceList extends Component {
+	render() {
+		const { preferences, translate, unsetPreference } = this.props;
+		return (
+			<div>
+				<a
+					href={ '/devdocs/client/state/preferences/README.md' }
+					title={ translate( 'Preferences' ) }
+				>
+					{ translate( 'Preferences' ) }
+				</a>
+				<Card className="preferences-helper__current-preferences">
+					{ ! isEmpty( preferences ) ? (
```

Comment:
If we do add query preferences, we also need to check that we're not currently fetching here by checking `isFetchingPreferences`.